wd_core.au3

```diff
@@ -1206,6 +1206,8 @@ Func _WD_Startup()
 
 	If $_WD_DRIVER = "" Then
 		Return SetError(__WD_Error($sFuncName, $_WD_ERROR_InvalidValue, "Location for Web Driver not set."), 0, 0)
+	ElseIf Not FileExists($_WD_DRIVER) Then
+		Return SetError(__WD_Error($sFuncName, $_WD_ERROR_FileIssue, "Web Driver not exist in location: " & $_WD_DRIVER), 0, 0)
```

Comment:
This error message needs some work --

- "Web Driver not exist" isn't proper English. To correct, you could change "not" to "does not" or "doesn't".
- An couple of alternativea would be `"Non-existent Web Driver: " & $_WD_DRIVER` or `"Missing Web Driver: " & $_WD_DRIVER`

Reply:
Take a look on: [rewording error message](https://github.com/Danp2/au3WebDriver/pull/273/commits/a105aeb26c048fa34be7eb9786361310b32f7479)